app/src/main/java/ua/dancecamp/vmedvediev/ua21summerdancecamp/UI/bottomnavigation/events/EventsFragment.kt

```diff
@@ -172,6 +185,16 @@ class EventsFragment : Fragment(), TabLayout.OnTabSelectedListener {
         }
     }
 
+    private fun updateTabs() {
+        namesOfDaysArray = emptyArray()
```

Comment:
What is going on here?

Reply:
I am updating tab language if it is changed. Renamed to updateTabsLanguage()

Reply:
No, I'm talking about ` namesOfDaysArray = emptyArray()` and nest line `namesOfDaysArray = context?.resources?.getStringArray(R.array.namesOfDaysArray)`
I can't understand what is going on here